### PR TITLE
Add a Render deployment guide

### LINKED ISSUE #556: User guide for Render.com to host Web Service or Database
**Which module(s) and week(s) does this change affect?**
Module(s): Node and Databases
Week(s): 1 to 3

**What is the work that needs to be done?**
[DBeaver_How to connect Render.com Postgres database.pdf](https://github.com/CodeYourFuture/syllabus/files/10335548/DBeaver_How.to.connect.Render.com.Postgres.database.pdf)
[Render.com_How to create a Postgres database.pdf](https://github.com/CodeYourFuture/syllabus/files/10335549/Render.com_How.to.create.a.Postgres.database.pdf)
[How to deploy a server on Render.pdf](https://github.com/CodeYourFuture/syllabus/files/10335550/How.to.deploy.a.server.on.Render.pdf)
[Render.com_How to connect your Github account.pdf](https://github.com/CodeYourFuture/syllabus/files/10335551/Render.com_How.to.connect.your.Github.account.pdf)

**Why is this work important to do?**
Heroku no longer offers a free plan at the moment.
This is a user guide to assist trainees who want to use Render.com to host a web service and database.

**Additional context**
These are the links to the user guides.

**How to connect your GitHub account to Render.com**
https://docs.google.com/document/d/1EZ_cU46cMnLr07UKTLh0Ol5oegqYPZqbR6hw1Pp2S2k/edit#heading=h.3qgf0u7hw1om

**How to deploy a Web Service on Render.com**
https://docs.google.com/document/d/1DgWG7-lgtQVDsffmdsJNNrM3M-wloCxOHQow2cB49qE/edit#heading=h.xgoy8sc8p1ku

**How to set up a database on Render.com**
https://docs.google.com/document/d/1Gdcx4dkIJznV2jLRlbwfQA4Udsh-Qa6MW4vdnOVmU04/edit

**Connecting DBeaver (or any other database management system) to the Database on Render.com**
https://docs.google.com/document/d/172jWo4emoWDTr9DBsE6RAnEYrp3E4JeGYPi19F-f5XY/edit

**Who might need to know about this change?**
@SallyMcGrath, @Dedekind561

### PR DESCRIPTION
**Pull request by alismith-dev**
## What does this change?

Module: Render deployment guide
Week(s): N/A

## Checklist

- [x] I have read the [contributing guidelines](https://syllabus.codeyourfuture.io/contributing/overview)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide)

## Description

Closes #556.

Adds a deployment guide for [Render](https://render.com/). I've made the odd working tweak here or there, but I've largely followed the content from #556.

## Who needs to know about this?

@CodeYourFuture/global-syllabus 
@textbook? Not sure if this is relevant for templating final projects?

### CODE REVIEW

File: docs/guides/deployment-render/connecting-a-dbms.md
Comment (alismith-dev) [inline docs/guides/deployment-render/connecting-a-dbms.md]: Is it worth flagging that I couldn't find any reference to DBeaver or other DB clients in our DB module?

In which case I did wonder if this guide was a distraction.

Comment (alismith-dev) [reply docs/guides/deployment-render/connecting-a-dbms.md]: I'm also now doubting if I've got the DBMS jargon correct here. I am but a simple frontend dev, so would appreciate corrections if this is incorrect.

**Comment by 40thieves** [comment]
Oh I also forgot to tag @ShaunCYFSA who created the original content 🙂 Thanks!

File: docs/guides/deployment-render/connecting-a-dbms.md
Comment (Dedekind561) [reply docs/guides/deployment-render/connecting-a-dbms.md]: Hey @40thieves ,
First 2 set of instructions work very well - I very quickly set up and deployed a web service and db.
I'm inclined to agree with your point on DBeaver - if we don't mention it, we should leave it up to them to figure out how to connect to their db.
However, we could put in a small note on using the command line as you can generate a CLI command using the external connection tab

**Comment by Dedekind561** [comment]
Also happy for this to go through now - DBMS guide won't do any harm

**Review by Dedekind561 — COMMENTED** [review]
Also, the links on the overview page don't work when you first click on them -  but they then start working when you click on the left hand side links 🤷 
Is this just me or a quirk of Docusaurus?

**Comment by gregdyke** [comment]
Docusaurus quirk

File: docs/guides/deployment-render/connecting-a-dbms.md
Comment (alismith-dev) [reply docs/guides/deployment-render/connecting-a-dbms.md]: > However, we could put in a small note on using the command line as you can generate a CLI command using the external connection tab

Do you mean connecting the postgres CLI to a Render-hosted instance?

Assuming yes, I could look into writing up some instructions for that but I'd have to test myself to know for sure (or if you have something to hand I'd be happy to add it). Maybe that could be done in a follow up?

**Comment by 40thieves** [comment]
I've rebased this on `master` after merging #624 and updated the wording of the DBeaver page as Keith pointed out (on Slack) that DBMS was indeed not the correct terminology.

Moving back for review.

File: docs/guides/deployment-render/connecting-a-dbms.md
Comment (Dedekind561) [reply docs/guides/deployment-render/connecting-a-dbms.md]: Yes exactly re postgres CLI to a Render-hosted instance.
And yeah I think we should do that in a follow up - let's get this in for now.

Comment (alismith-dev) [reply docs/guides/deployment-render/connecting-a-dbms.md]: Thanks - could you give this a 👍🏻 so I can merge it?

**Review by Dedekind561 — APPROVED** [review]

**Pull request merged**
